readme_us.md

```diff
@@ -1,51 +1,52 @@
-<img align="left" width="80" src="https://i.imgur.com/AbYOj2T.png">
+# readme
 
-# ggram – Telegram Bot Framework
+![https://i.imgur.com/AbYOj2T.png](https://i.imgur.com/AbYOj2T.png)
 
-> 🇬🇧 🇺🇸 NEED TRANSLATORS. Please, make pull requests if you wanna help. Temporarily I recommend [deepl](https://www.deepl.com/translator) to translate [this page](https://raw.githubusercontent.com/TRIGONIM/ggram/main/readme.md)
+# ggram – Telegram Bot Framework
 
-<p align="left">
-	<img src="https://img.shields.io/github/languages/code-size/TRIGONIM/ggram">
-	<img src="https://img.shields.io/github/license/TRIGONIM/ggram">
-</p>
+> 🇬🇧 🇺🇸 NEED TRANSLATORS. Please, make pull requests if you wanna help. Temporarily I recommend deepl to translate this page
+> 
 
-Создавайте Telegram ботов любой сложности и **запускайте их на Garry's Mod сервере**
+[https://img.shields.io/github/languages/code-size/TRIGONIM/ggram](https://img.shields.io/github/languages/code-size/TRIGONIM/ggram)
 
-<img align="right" width="300" src="https://user-images.githubusercontent.com/9200174/135781831-dbb545a9-b3d9-4d0a-ba58-dd42935d35f0.png">
+[https://img.shields.io/github/license/TRIGONIM/ggram](https://img.shields.io/github/license/TRIGONIM/ggram)
 
-```lua
-local bot = ggram("token")
+Create Telegram bots any difficulty and start it on your own **Garry’s Mod server**
 
-bot.enable_polling() -- enables getUpdates loop
+![https://user-images.githubusercontent.com/9200174/135781831-dbb545a9-b3d9-4d0a-ba58-dd42935d35f0.png](https://user-images.githubusercontent.com/9200174/135781831-dbb545a9-b3d9-4d0a-ba58-dd42935d35f0.png)
 
-bot.command("start", function(ctx)
-	ctx.reply.text("Hello @" .. ctx.from.username)
-end)
 ```
+local bot = ggram("token")bot.enable_polling() -- enables getUpdates loopbot.command("start", function(ctx)    ctx.reply.text("Hello @" .. ctx.from.username)end)
+```
+
+Extra examples can be found in directory [/bots](/lua/ggram/bots)
 
-Дополнительные примеры можно найти в папке [/bots](/lua/ggram/bots)
+## 🔥 Start of using
 
-## 🔥 Начало использования
-0. У вас уже должен быть создан бот через [@BotFather](https://t.me/BotFather)
-1. Скачайте ggram с этого репозитория и установите в `/addons/ggram`
-2. Создайте файл `/addons/ggram_bots/lua/ggram/bots/anyname/_init.lua`
-3. Заполните его содержимым демонстративного бота из папки [/bots/example](/lua/ggram/bots/example), заменив токен
+1. You should have already created a bot using [@BotFather](https://t.me/BotFather).
+2. Download ggram from this repository and install it in `/addons/ggram'.
+3. Create a file `/addons/ggram_bots/lua/ggram/bots/anyname/_init.lua`.
+4. Fill it with bot example from [/bots/example](notion://www.notion.so/lua/ggram/bots/example), replacing token
```

Comment:
Copy the code of the example bot from [/bots/example](notion://www.notion.so/lua/ggram/bots/example) into the file you just created. Do not forget to replace the token.